tests/unit/mock-b2g-bin.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+//
+// Simple server to mock up a Firefox debug service for startSimulator()
+//
+// The tested code only needs a successful connection to work, and the testing
+// code makes another connection to introspect the command options used to
+// launch the mock simulator
+//
+
+var net = require('net');
+
+// This server will self-destruct after 1.5 seconds...
+// HACK: It's a race condition, but detecting a server should ideally happen
+// faster than this. Travis can be slow sometimes, though.
+var serverLifetime = 3000;
+
+// Quick & filthy command line options parsing
+var options = {
+  'binary': process.argv[1],
```

Comment:
The *simulator* uses command line args, and this is mocking that up so that it understands the options `startSimulator` uses to launch an FxOS simulator.